criu/cgroup.c

```diff
@@ -52,6 +53,8 @@ static u32 cg_set_ids = 1;
 static LIST_HEAD(cgroups);
 static unsigned int n_cgroups;
 
+struct mount_info *host_mountinfo;
```

Comment:
pls write a comment why we need this global variable

Reply:
done.